faf-java-server-app/src/main/java/com/faforever/server/coop/CoopService.java

```diff
@@ -30,6 +31,10 @@ public void reportOperationComplete(Player player, boolean secondaryTargets, Dur
     Requests.verify(game != null, ErrorCode.COOP_CANT_REPORT_NOT_IN_GAME);
 
     log.debug("Player '{}' reported coop result '{}' with secondary targets '{}' for game: {}", player, duration, secondaryTargets, game);
+    if (game.getValidity() != Validity.COOP_UNRANKED) {
+        log.debug("Game is not valid");
```

Comment:
Bad logging statement. If i look at the log this statement will not provide any information to me. Log which game and that it is a ccop game.